docs/dev-tools/chopsticks/overview.md

```diff
@@ -399,10 +382,8 @@ These are the methods that can be invoked and their parameters:
 ??? function "**dev_timeTravel** (date) — Sets the timestamp of the block to a specific date"
 
     === "Parameter"
-        
-        | Name  |       Type       |                                               Description                                                |
-        | :---: | :--------------: | :------------------------------------------------------------------------------------------------------: |
-        | date  | string \| number | Timestamp or date string to set. All future blocks will be sequentially created after this point in time |
+
+        - `date` (*string \| number*) - timestamp or date string to set. All future blocks will be sequentially created after this point in time
```

Comment:
the backslash after `string` is just so that it can be properly formatted in a table. Otherwise, when you run the formatter on tables that use `|` in one of the cells, it actually thinks you want to add another column. So since we've moved this out of the table format, we no longer need it here